### ISSUE TITLE
π₀ performance on RoboTwin 2.0

### ISSUE THREAD

**Issue by ShaolongLi616** [PRIME-RL/SimpleVLA-RL#54]
Thanks for releasing the code and paper.

I noticed in your paper you report π₀ achieves much higher success rate on some RoboTwin 2.0 tasks than public leaderboard (https://robotwin-platform.github.io/leaderboard). For example, in your Table 4, π₀ is reported to have 59% success on the Beat Block Hammer task (random). However, on the RoboTwin public leaderboard, π₀’s success rate on Beat Block Hammer seems to be only around 21%.

Could you please share the config you used for π₀ in that comparison, including hyperparameters (learning rate, batch size, etc.) and training setup?
